developers/deployments/addresses.md

```diff
@@ -11,37 +11,49 @@
+0xe8537a3d056da446677b9e9d6c5db704eaab4787">0xe8537a3d056da446677b9e9d6c5db704eaab4787</a></td><td><a href="https://explorer.celo.org/mainnet/address/0x434563B0604BE100F04B7Ae485BcafE3c9D8850E">0x434563B0604BE100F04B7Ae485BcafE3c9D8850E</a></td></tr><tr><td>StableTokenXOF</td><td><a href="https://explorer.celo.org/mainnet/address/0x73F93dcc49cB8A239e2032663e9475dd5ef29A08">0x73F93dcc49cB8A239e2032663e9475dd5ef29A08</a></td><td><a href="https://explorer.celo.org/mainnet/address/0x434563B0604BE100F04B7Ae485BcafE3c9D8850E">0x434563B0604BE100F04B7Ae485BcafE3c9D8850E</a></td></tr><tr><td>StableTokenKES</td><td><a href="https://explorer.celo.org/mainnet/address/
+0x7242a5FfbC765F322ee7cAd7957c291aC54c33F4">0x7242a5FfbC765F322ee7cAd7957c291aC54c33F4</a></td><td><a href="https://explorer.celo.org/mainnet/address/0x434563B0604BE100F04B7Ae485BcafE3c9D8850E">0x434563B0604BE100F04B7Ae485BcafE3c9D8850E</a></td></tr>
+<tr><td>StableTokenPSO</td><td><a href="https://explorer.celo.org/mainnet/address/
+0x456a3D042C0DbD3db53D5489e98dFb038553B0d0">0x456a3D042C0DbD3db53D5489e98dFb038553B0d0</a></td><td><a href="https://explorer.celo.org/mainnet/address/0x434563B0604BE100F04B7Ae485BcafE3c9D8850E">0x434563B0604BE100F04B7Ae485BcafE3c9D8850E</a></td></tr>
```

Comment:
this looks wrong, address points to cKES [0x456a3D042C0DbD3db53D5489e98dFb038553B0d0](https://celoscan.io/address/0x456a3D042C0DbD3db53D5489e98dFb038553B0d0#code)